translations/README.chinese.md

```diff
@@ -0,0 +1,215 @@
+    - [远程模式](#远程模式)
+    - [通过数据库事件调用Webhook](#在数据库事件上触发Webhooks)
+- [演示版](#演示版)
+    - [实时应用](#实时应用)
+    - [影片](#影片)
+- [支持与故障排除](#支持与故障排除)
+- [参与发展](#贡献)
+- [品牌资产](#品牌资产)
+- [执照](#执照)
+- [翻译](#翻译)
+
+<!-- markdown-toc end -->
+
+## 快速开始：
+
+### One-click deployment on Heroku
+
+尝试Hasura最快的方法是通过Heroku。
+
+1. 单击以下按钮，在带有免费Postgres附加组件的Heroku上部署GraphQL Engine:
+
+    [![部署到Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/hasura/graphql-engine-heroku)
+
+2. 打开Hasura控制台
+
+   访问 `https://<app-name>.herokuapp.com` (*更换 \<应用名称\> 使用您的应用名称*) 打开管理控制台.
+
+3. 进行第一个GraphQL查询
+
+### 其他一键式部署选项
+
+查看有关以下一键式部署选项的说明：
+
+| **基础设施提供商** | **一键式链接** | **附加信息** |
+|:------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------:|
+| DigitalOcean | [![部署到 DigitalOcean](https://graphql-engine-cdn.hasura.io/img/create_hasura_droplet_200px.png)](https://marketplace.digitalocean.com/apps/hasura?action=deploy&refcode=c4d9092d2c48&utm_source=hasura&utm_campaign=readme) | [docs](https://docs.hasura.io/1.0/graphql/manual/guides/deployment/digital-ocean-one-click.html#hasura-graphql-engine-digitalocean-one-click-app) |
+| Azure | [![部署到 Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3a%2f%2fraw.githubusercontent.com%2fhasura%2fgraphql-engine%2fmaster%2finstall-manifests%2fazure-container-with-pg%2fazuredeploy.json) | [docs](https://docs.hasura.io/1.0/graphql/manual/guides/deployment/azure-container-instances-postgres.html) |
+
+### 其他部署方法
+
+有关基于Docker的部署和高级配置选项，请参阅[部署
+指南](https://docs.hasura.io/1.0/graphql/manual/getting-started/index.html) 要么
+[安装清单](../install-manifests).
+
+## 建筑
+
+Hasura GraphQL引擎位于Postgres数据库实例的前面，并且可以接受来自客户端应用程序的GraphQL请求。 可以将其配置为与现有的身份验证系统一起使用，并可以使用具有来自身份验证系统的动态变量的字段级规则来处理访问控制。
+
+您还可以合并远程GraphQL模式并提供统一的GraphQL API。
+
+![Hasura GraphQL引擎架构](../assets/hasura-arch.svg)
+
+## 客户端工具
+
+Hasura可与任何GraphQL客户端一起使用。 我们建议使用[Apollo Client](https://github.com/apollographql/apollo-client). 见[awesome-graphql](https://github.com/chentsulin/awesome-graphql) 获取客户列表.
+
+## 添加业务逻辑
+
+GraphQL Engine提供了易于使用，可扩展和高性能的方法，用于向您的后端添加自定义业务逻辑:
+
+### 远程模式
+
+除了Hasura基于Postgres的GraphQL模式之外，还可以在远程模式中添加自定义解析器。 适用于实现支付API或查询数据库中不存在的数据等用例 - [阅读更多](../remote-schemas.md).
+
+### 在数据库事件上触发Webhooks
+
+添加基于数据库事件触发的异步业务逻辑。
+通知，Postgres或异步数据管道的理想选择
+处理中 - [阅读更多](../event-triggers.md).
+
+### 派生数据或数据转换
+
+在Postgres中转换数据或在其上运行业务逻辑以导出可以使用GraphQL Engine查询的另一个数据集 - [阅读更多](https://docs.hasura.io/1.0/graphql/manual/queries/derived-data.html).
+
+## 演示版
+
+在中查看所有示例应用程序
+[community/examples](../community/examples) 目录.
```

Comment:
This link is not working. It was broken on `master` as well. It will probably be fixed when you merge master 🙂